model_api/python/model_api/adapters/utils.py

```diff
@@ -527,5 +529,5 @@ def __call__(self, inputs):
         if self.is_trivial:
             return inputs
         if self.reverse_input_channels:
-            inputs = cv2.cvtColor(inputs, cv2.COLOR_BGR2RGB)
+            inputs = inputs[...,::-1]
```

Comment:
I'd use cv2 for images (perhaps it's faster)

Reply:
I changed it for processing staked images because previous code can only single image at a time.
Are there other reason to use cv2 I missed?

Reply:
That's performance, we don't know if numpy is faster or slower

Reply:
I tried to compare speed using code below
```
import cv2
import numpy as np
import time

IMAGE_PATH = "data/coco/train2017/000000149165.jpg"

def run_cv(img: np.array):
    res = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
    return res

def run_np(img: np.array):
    res = img[..., ::-1]
    return res

if __name__ == "__main__":
    img = cv2.imread(IMAGE_PATH)

    start_time = time.perf_counter()
    for _ in range(10000000):
        # run_np(img)
        run_cv(img)
    print(time.perf_counter() - start_time)
    
```
and result is 

- cv : 1.1237245709635317
- numpy : 0.017020812025293708

It seems that numpy is faster than cv2. How about using numpy?

Reply:
Could you re-run the test for a bigger resolution to avoid the case when numpy does just an index remapping without changing the memory layout? The time should grow linearly with the amount memory to restructure.

Reply:
I don't know the reason, but speed changes a lot so I tested original resolution once again w/ code below.
```Python
import cv2
import numpy as np
import time

IMAGE_PATH = "data/coco/train2017/000000149165.jpg"

def run_cv(img: np.array):
    res = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
    return res

def run_np(img: np.array):
    res = img[..., ::-1]
    return res

if __name__ == "__main__":
    img = cv2.imread(IMAGE_PATH)
    # img = cv2.resize(img, (4096, 4096))

    start_time = time.perf_counter()
    for _ in range(10000):
        run_np(img)
        # run_cv(img)
    print(time.perf_counter() - start_time)
```


Here is the result w/ (612, 612, 3)

- cv : 0.119686498073861
- numpy : 0.0037762820720672607

these are result w/ (4096, 4096, 3)

- cv : 295.947567245923
- numpy : 0.119686498073861

Reply:
I see, that's because numpy actually uses mapping rather than true reordering.
Please, use `res = np.ascontiguousarray(img[..., ::-1])`

Reply:
I checked that it spends 555.3918372320477 seconds. It's slower than cv2 if forcing reordering.
I don't know well about this part, but is it necessary to force actual reordering?

Reply:
reverted it. https://github.com/openvinotoolkit/model_api/pull/178/commits/4708d2bcbd61e6d1e911161132af885cbdab210f